### PR TITLE
fix #2465

### LINKED ISSUE #2465: accounts activity has a separate network check instead of using the function from utils
**Actual Behaviour**

accounts activity has a separate network check instead of using the function from utils.

**Expected Behaviour**

the function from utils.java should be used.

**Steps to reproduce it**

open accountsactivity.java and navigate to onItemClick. a separate network check instead of using the function from utils class.

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by kk63852**
Fixed #2465 

Changes: 
separate network check was replaced with the function from utils class.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/utilities/Utils.java [hunk 1]
```diff
@@ -167,7 +169,7 @@ public static boolean checkNetwork(Context context, @NonNull View view) {
         if (isInternetOn(context)) {
             return true;
         } else
-            SnackBarHandler.show(view,context.getString(R.string.not_connected));
+            Toast.makeText(context,R.string.internet_is_off,Toast.LENGTH_SHORT).show();
```
Comment (yashk2000) [inline app/src/main/java/org/fossasia/phimpme/utilities/Utils.java]: Why was this SnackBar changed to a toast?

Comment (kk63852) [reply app/src/main/java/org/fossasia/phimpme/utilities/Utils.java]: because this snackbar was not showing when checknetwork returned false.

**Comment by yashk2000** [comment]
> Pls don't remove the snack bar and instead try to make it appear. Please check where the bug is

@abishekvashok I have opened #2467 and #2470 to deal with the snackbar position and visibility issues and am working on them.

**Comment by kk63852** [comment]
@yashk2000 please exclude this snackbar from your pr since i have already opened the pr, i would resolve this snackbar issue.

**Comment by yashk2000** [comment]
@kk63852 the change I will make might affect a lot of snackbars. But I will try my best to leave any changes pertaining to this snackbar.

**Comment by kk63852** [comment]
@yashk2000 thanks.

**Comment by yashk2000** [comment]
@kk63852 at the place where you've used the toast, replace the toast with a proper snackbar created by you there instead of calling the Snackbarhandler. That way changes made by me will not affect changes in this pull request. Is it ok to do it this way?

**Comment by kk63852** [comment]
@yashk2000 checkout my latest commit.

**Comment by yashk2000** [comment]
LGTM

File: app/src/main/java/org/fossasia/phimpme/utilities/Utils.java [hunk 2]
```diff
@@ -15,11 +15,14 @@
 import android.provider.MediaStore;
 import android.speech.RecognizerIntent;
 import android.support.annotation.NonNull;
+import android.support.design.widget.Snackbar;
 import android.util.Base64;
 import android.view.View;
 import android.webkit.MimeTypeMap;
+import android.widget.Toast;
```
Comment (yashk2000) [inline app/src/main/java/org/fossasia/phimpme/utilities/Utils.java]: I think this import will be unused now. If it is unused remove it.

**Comment by abishekvashok** [comment]
Thanks

**Pull request merged**